agent/recorder-agent.c

```diff
@@ -128,70 +118,16 @@ static gint64
 hwangsae_recorder_agent_start_recording (HwangsaeRecorderAgent * self,
     gchar * edge_id)
 {
-  g_autoptr (GError) error = NULL;
-  g_autofree gchar *host = g_strdup (self->relay_address);
-  guint port = self->relay_stream_port;
-  g_autofree gchar *streamid_tmp = NULL;
-  g_autofree gchar *streamid = NULL;
-  g_autofree gchar *url = NULL;
-  g_autofree gchar *recording_edge_dir = NULL;
-  g_autofree gchar *filename_prefix = NULL;
-
-  if (self->is_recording) {
-    g_warning ("recording already started");
-    return self->recording_id;
-  }
-
-  self->is_recording = TRUE;
-  self->recording_id = g_get_real_time ();
-
-  hwangsae_recorder_agent_send_rest_api (self, RELAY_METHOD_START_STREAMING,
-      edge_id);
-
-  streamid_tmp = g_strdup_printf ("#!::r=%s,u=%s", edge_id, self->recorder_id);
-  streamid = g_uri_escape_string (streamid_tmp, NULL, FALSE);
-  url = g_strdup_printf ("srt://%s:%d?streamid=%s", host, port, streamid);
-
-  g_debug ("starting to recording stream from %s", url);
-
-  g_autofree gchar *recording_dir = g_strdup (self->recording_dir);
-
-  if (g_str_equal (recording_dir, "")) {
-    g_free (recording_dir);
-    recording_dir = g_build_filename (g_get_user_data_dir (),
-        "hwangsaeul", "hwangsae", "recordings", NULL);
-  }
-
-  recording_edge_dir = g_build_filename (recording_dir, edge_id, NULL);
+  g_debug ("Default implementation");
 
-  g_debug ("setting recording_dir: %s\n", recording_edge_dir);
-
-  filename_prefix = g_strdup_printf ("hwangsae-recording-%ld",
-      self->recording_id);
-
-  hwangsae_recorder_set_recording_dir (self->recorder, recording_edge_dir);
-
-  hwangsae_recorder_set_filename_prefix (self->recorder, filename_prefix);
-
-  hwangsae_recorder_start_recording (self->recorder, url);
-
-  return self->recording_id;
+  return 0;
 }
 
 static void
 hwangsae_recorder_agent_stop_recording (HwangsaeRecorderAgent * self,
     gchar * edge_id)
 {
-  g_autoptr (GError) error = NULL;
-
-  if (!self->is_recording) {
-    g_warning ("recording already stopped");
-    return;
-  }
-
-  self->is_recording = FALSE;
-
-  hwangsae_recorder_stop_recording (self->recorder);
+  g_debug ("Default implementation");
```

Comment:
You should probably switch the executable of `hwangsae-recorder.service` to `hwangsae-simple-recorder-agent` until the new default agent gets fully implemented.

Reply:
Oh, you are right. I totally missed that part. Thanks for pointing that.